tests/masonry/builders/test_complete.py

```diff
@@ -507,7 +507,7 @@ def test_package_with_include(mocker):
             "src_package",
             "tests",
         ]
-        assert ns["package_data"] == {"": ["*"]}
+        assert ns["package_data"] == {"": ["*"], "extra_dir": ["vcs_excluded.txt"]}
```

Comment:
I am unsure if this behaviour is actually desired. Since `extra_dir` is not necessarily a package. According to the [`setuptools` documenation](https://setuptools.readthedocs.io/en/latest/setuptools.html), this needs to be a package name.